### PR TITLE
home page created

### PR DESCRIPTION
**Pull request by aishwarya-bm**
1. Home page created
2. added skeleton for other html pages

home

### CODE REVIEW

File: screens/gamelevel.html [hunk 1]
```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Quiz | Start Game</title>
+    <link rel="stylesheet" href="../style.css">
+
```
Comment (shuntzhere) [inline screens/gamelevel.html]: Put the title below the stylesheet.

File: screens/question.html [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Quiz | Question</title>
+    <link rel="stylesheet" href="../style.css">
+</head>
```
Comment (shuntzhere) [inline screens/question.html]: Same put title below.

**Review by shuntzhere — APPROVED** [review]
The code is very clean and the classes are self-explained. Suggested some minor changes. Rest looks good to me.

File: screens/gamelevel.html (hunk 1 above)
Comment (aishwarya-bm) [reply screens/gamelevel.html]: done

File: screens/question.html (hunk 2 above)
Comment (aishwarya-bm) [reply screens/question.html]: done

File: screens/gamelevel.html [hunk 3]
```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="../style.css">
+    <title>Quiz | Start Game</title>
+
```
Comment (tanishq20) [inline screens/gamelevel.html:10]: Avoid giving extra spaces

File: screens/gamelevel.html [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+                    <ul class="right-nav d-flex">
+                        <li class="list-no-bullet"><a target="_blank" href=""><i>Hi, Aishwarya</i></a>
+                        </li>
+                        <li class="list-no-bullet"><a target="_blank" href="">logout</span></a>
+                        </li>
+                    </ul>
+                </nav>
+            </div>
+        </nav>
+        <br>
```
Comment (tanishq20) [inline screens/gamelevel.html:33]: Avoid using `br` tag

File: screens/home.html [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+                        </li>
+                        <li class="list-no-bullet"><a target="_blank" href="">
+                                <i class="fa-solid fa-arrow-right-from-bracket"></i>
+                                </span></a>
+                        </li>
+                    </ul>
+                </nav>
+            </div>
+        </nav>
+        <br>
```
Comment (tanishq20) [inline screens/home.html:34]: Same here

File: screens/home.html [hunk 6]
```diff
@@ -0,0 +1,111 @@
...
+
+        <main>
+            <h4 class="page-heading text-center">Choose a category</h4>
+            <ul class="category-container list-no-bullet">
+                <li class="list-item cat-item">
```
Comment (tanishq20) [inline screens/home.html]: `cat-item` can be written as `category-item` for better readability.

**Review by tanishq20 — COMMENTED** [review]
Code looks clean. Suggested some changes.

File: screens/gamelevel.html (hunk 3 above)
Comment (aishwarya-bm) [reply screens/gamelevel.html:10]: done

File: screens/gamelevel.html (hunk 4 above)
Comment (aishwarya-bm) [reply screens/gamelevel.html:33]: done

File: screens/home.html (hunk 6 above)
Comment (aishwarya-bm) [reply screens/home.html]: renamed

**Review by tanishq20 — APPROVED** [review]

**Pull request merged**